traffic_ops/app/lib/MojoPlugins/Health.pm

```diff
@@ -159,6 +159,9 @@ sub register {
 				cachegroups  => [],
 			};
 
+			$self->app->log->debug("get_cache_health() - " . 
```

Comment:
can you remove this? we don't typically leave debug statements in the code. thanks!